addons/dex-k8s-authenticator/1.1.x/dex-k8s-authenticator-1.yaml

```diff
@@ -30,7 +30,7 @@ spec:
   chartReference:
     chart: dex-k8s-authenticator
     repo: https://mesosphere.github.io/charts/staging
-    version: 1.1.12
+    version: 1.1.13
```

Comment:
We need to make an entirely new revision of dex for this, we can no longer change these inline (but note, we're looking at ways to [improve this process in future versions](https://jira.mesosphere.com/browse/DCOS-62943)).

Reply:
The way I have understood is that there isn't a release happened out of this new structure. We would create a new revision once the release is out. cc @dkoshkin

Reply:
@GoelDeepak Thats how I understood it too, since we didn't have a release yet that it would be fine to change in-place, but @shaneutt might have more context.